rest/data/buildlogger.go

```diff
@@ -188,6 +188,7 @@ func (dbc *DBConnector) FindLogsByTestName(ctx context.Context, opts Buildlogger
 		paginated bool
 	)
 
+	opts.Group = ""
```

Comment:
What's the significance of clearing the group for the data connectors?

Reply:
Just in case it was set, it shouldn't be set here. Sanity thing

Reply:
Sure, but why is it invalid for it to be set here?

Reply:
cause the test name doesn't allow filtering by group (there is a special group route for that)